package.json

```diff
@@ -108,7 +108,8 @@
     "fmt": "prettier --write .*.json *.{json,md,js} **/*.ts './{.circleci,.github,build,doc,lib,locales,test,typings}/**/*.{json,md,js,ts,html}'",
```

Comment:
This is not a valid call to `prettier` and results in the usage output:

```
> prettier --write

Usage: prettier [options] [file/dir/glob ...]

By default, output is written to stdout.
Stdin is read if it is piped to Prettier and no files are given.
```

Reply:
This will lint things we haven't been linting before like CSS, YAML, and XHTML files. I see no reason not to be linting those though, but just wanted to document that.